### PR TITLE
Update security/authentication/azure-active-directory/index.md

### LINKED ISSUE #6805: Add Azure AD samples to TOC
* [Integrating Azure AD V2 into an ASP.NET Core web app](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2)
* [Calling a ASP.NET Core Web API from a WPF application using Azure AD V2](https://github.com/azure-samples/active-directory-dotnet-native-aspnetcore-v2)

See `More ASP.NET AAD samples, and 2 on-demand videos about Azure AD V2.` email.

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #6805 
@jmprieur please review.

### CODE REVIEW

File: aspnetcore/security/authentication/azure-active-directory/index.md [hunk 1]
```diff
@@ -17,3 +17,11 @@ uid: security/authentication/azure-active-directory/index
 * [Calling a Web API in an ASP.NET Core Web Application Using Azure AD](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-webapi-openidconnect-aspnetcore/)
```
Comment (jmprieur) [inline aspnetcore/security/authentication/azure-active-directory/index.md]: Arguably the articles are really the manifestation of samples (for instance for the first one: https://github.com/Azure-Samples/active-directory-dotnet-webapp-openidconnect-aspnetcore)

Comment (jmprieur) [inline aspnetcore/security/authentication/azure-active-directory/index.md]: @Rick-Anderson, I wonder if you don't want to have:
## Azure AD V1 samples
Then the first 2 samples that had initially

## Azure AD B2C sample
The 4th sample

## Azure AD V2 samples
What you just added

Do you want me to send a PR? (but that would not be before Monday)à

Comment (Rick-Anderson) [reply aspnetcore/security/authentication/azure-active-directory/index.md]: > Do you want me to send a PR? 

Next week would be great.

Comment (jmprieur) [reply aspnetcore/security/authentication/azure-active-directory/index.md]: ok will do.

**Pull request closed without merging**